.simplecov

```diff
@@ -1,7 +1,7 @@
 SimpleCov.start do
-  track_files 'lib/**/*.rb'
```

Comment:
See https://github.com/troessner/reek/issues/1419#issuecomment-422386621 on why this line is needed.

Reply:
Re-added.